cypress/tsconfig.json

```diff
@@ -4,9 +4,7 @@
     "baseUrl": "../node_modules",
     "target": "es5",
     "lib": ["es5", "dom"],
-    "types": ["cypress"],
+    "types": ["cypress"]
   },
-  "include": [
-    "**/*.ts"
-  ]
+  "include": ["**/*.ts", "integration/sample.spec.js"]
```

Comment:
did you intend to put sample.spec.js in here?

Reply:
ah, no :) Will remove.